src/openforms/registrations/contrib/zgw_apis/utils.py

```diff
@@ -0,0 +1,36 @@
+from datetime import date, datetime
+
+import pytz
+
+
+def process_according_to_eigenschap_format(specificatie: dict, value: str):
+    """
+    This is an extra validation we are doing against the eigenschap.specificatie.formaat
+    since the value we send should be YYYYMMDD for date or YYYYMMDDHHmmSS for date/time.
+    """
+    format = specificatie.get("formaat")
+    if format not in ["datum", "datum_tijd"]:
+        return value
+
+    if format == "datum":
+        try:
+            valid_date = date.fromisoformat(value)
+        except ValueError as exc:
+            raise exc
+
+        processed_value = valid_date.strftime("%Y%m%d")
+
+    elif format == "datum_tijd":
+        try:
+            valid_datetime = datetime.fromisoformat(value)
+        except ValueError as exc:
+            raise exc
+
+        amsterdam_timezone = pytz.timezone("Europe/Amsterdam")
```

Comment:
you can re-use `openforms.contrib.clients.utils.TIMEZONE_AMS` instead :thinking: 

It would probably be useful to have a helper function in that module:

```py
from django.utils import timezone

def datetime_in_amsterdam(value: datetime) -> datetime:
    return timezone.make_naive(value, timezone=TIMEZONE_AMS)

def get_today() -> str:
    now = datetime_in_amsterdam(timezone.now())
    return now.date().isoformat()
```